src/vast_tracker.js

```diff
@@ -597,46 +605,51 @@ export class VASTTracker extends EventEmitter {
    * @param {Object} [options={}] - An optional Object of options to be used in the tracking calls.
    */
   trackURLs(URLTemplates, macros = {}, options = {}) {
+    //Avoid mutating the object received in parameters.
+    const givenMacros = { ...macros };
     if (this.linear) {
       if (
         this.creative &&
         this.creative.mediaFiles &&
         this.creative.mediaFiles[0] &&
         this.creative.mediaFiles[0].fileURL
       ) {
-        macros['ASSETURI'] = this.creative.mediaFiles[0].fileURL;
+        givenMacros['ASSETURI'] = this.creative.mediaFiles[0].fileURL;
       }
       if (this.progress) {
-        macros['ADPLAYHEAD'] = this.progressFormatted();
+        givenMacros['ADPLAYHEAD'] = this.progressFormatted();
       }
     }
     if (this.creative?.universalAdIds?.length) {
-      macros['UNIVERSALADID'] = this.creative.universalAdIds
-        .map(universalAdId => universalAdId.idRegistry.concat(' ', universalAdId.value))
-        .join(',');
```

Comment:
Why removing the join here ? 
I see the specs saying "Multiple UniversalAdIDs can be supported with a comma separating them."

Reply:
Good catch! 👍 It was a fail during the merge conflict resolution. Resolved in https://github.com/dailymotion/vast-client-js/pull/382/commits/caea3e9dd63f5e78fe8a678314b1ff50d544ffcf